src/screen/SubmitTipScreen.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import {Button, Card, Icon, Input, Layout, ListItem, Text, TopNavigationAction} from '@ui-kitten/components';
+import ScreenNavigation from 'layout/ScreenNavigation';
+import globalStyles, {standardPadding} from 'src/styles';
+import {NavigationProp} from '@react-navigation/native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {ActivityIndicator, StyleSheet, View} from 'react-native';
+import {useContext, useReducer} from 'react';
+import {AccountContext} from 'context/AccountContextProvider';
+import {useAsyncRetry} from 'react-use';
+import {getAccountsIdentityInfo} from 'service/api/account';
+import {ChainApiContext} from 'context/ChainApiContext';
+import {u8aToString} from '@polkadot/util';
+import Identicon from '@polkadot/reactnative-identicon';
+import Padder from 'presentational/Padder';
+import {TxContext} from 'context/TxContext';
+import {useQueryClient} from 'react-query';
+
+export function SubmitTipScreen({navigation}: {navigation: NavigationProp<DashboardStackParamList>}) {
+  const {loading, value: data, error} = useAccount();
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const {start} = useContext(TxContext);
+  const {api} = useContext(ChainApiContext);
+  const queryClient = useQueryClient();
+
+  if (loading) {
+    return <ActivityIndicator />;
+  }
+
+  if (error || !data) {
+    return <Text>Something went wrong</Text>;
+  }
+
+  const account = data[0];
+  if (!account) {
+    return <Text>Account not found in this server</Text>;
+  }
+
+  const valid = state.beneficiary && state.reason && state.reason.length > 4;
+
+  return (
+    <Layout style={globalStyles.flex}>
+      <ScreenNavigation
+        renderTitle={() => (
+          <Text category={'s1'} style={globalStyles.monoFont}>
+            Submit Tip
+          </Text>
+        )}
+        accessoryLeft={
+          <TopNavigationAction onPress={navigation.goBack} icon={(p) => <Icon {...p} name={'arrow-back-outline'} />} />
+        }
+      />
+      <SafeAreaView edges={['bottom']} style={styles.container}>
```

Comment:
Shouldn't this `SafeAreaView` be the most outside wrapper?

Reply:
`ScreenNavigation.tsx` already has a safeAreaView in it, so we can't wrap that with another `SafeAreaView`. I can refactor that in another PR to have only one SafeAreaView per page. what do you think?

Reply:
It sounds good.